tests/test_helpers/test_theme.py

```diff
@@ -0,0 +1,50 @@
+"""Tests for theme helpers."""
+import pytest
+from django.contrib.auth import get_user_model
+from django.test import RequestFactory
+
+from futurex_openedx_extensions.helpers import theme
+
+
+@pytest.mark.django_db
+def test_get_fx_dashboard_url(base_data):  # pylint: disable=unused-argument
+    """Verify _get_fx_dashboard_url reurns correct address"""
+    factory = RequestFactory()
+    request = factory.get('/some-path')
+    request.site = type('Site', (), {})()
+    request.site.name = 's2.sample.com'
+    request.LANGUAGE_CODE = 'en'
+    # User with current site tenant access
```

Comment:
please add test case for `user1` to test system-staff access. Also use `parametrize` to send the user id and the expected URL instead of repeating the check

Another useful hint: instead of adding comments like `User with current site tenant access`, you can add the description on the assert failure like this:
```python
# error_message to be sent along with the ID and the expected URL. Something like "wrong URL returned. User has role access to the site"
assert url == expected_url, error_message
```